bin/i18n/resources/pegasus/emails/sync_in.rb

```diff
@@ -0,0 +1,48 @@
+#!/usr/bin/env ruby
+
+require 'fileutils'
+
+require_relative '../../../i18n_script_utils'
+require_relative '../../../utils/pegasus_markdown'
+require_relative '../emails'
+
+module I18n
+  module Resources
+    module Pegasus
+      module Emails
+        class SyncIn < I18n::Utils::SyncInBase
+          LOCALIZABLE_FILE_SUBPATHS = %w[
+            hoc_signup_2023_receipt_en.md
+          ].freeze
+
+          def execute
+            progress_bar.start
+
+            I18nScriptUtils.process_in_threads(LOCALIZABLE_FILE_SUBPATHS) do |file_subpath|
+              origin_file_path = File.join(ORIGIN_DIR_PATH, file_subpath)
+              next unless File.exist?(origin_file_path)
+
+              i18n_source_file_path = File.join(I18N_SOURCE_DIR_PATH, file_subpath)
+              i18n_source_file_path = i18n_source_file_path.sub('public/', '')
+
+              I18nScriptUtils.copy_file(origin_file_path, i18n_source_file_path)
+              I18n::Utils::PegasusMarkdown.sanitize_file_header(i18n_source_file_path)
```

Comment:
I don't remember by heart what exactly needed to be changed in other markdown headers, but I would double check we need to apply that `sanitize_file_header` method to the emails.

Reply:
Oh like to skip translating the `to:`, `from:`, etc. email sections since those will be the same for all translations of the email?

Reply:
So, the method `sanitize_file_header` calls `sanitize_header` and then it "slices" the field `Title`, you can read some more details in the [file comments](https://github.com/code-dot-org/code-dot-org/blob/staging/bin/i18n/utils/pegasus_markdown.rb) because I can't tell what exactly is doing.

Moreover, as you pointed out, these emails also have content we would want to prevent from being translated such as:
```
---
from: "blahblah@email.email"
reply-to: noreply@email.email
subject: "You're signed up for Hour of Code!"
---
  <% hourofcode = CDO.canonical_hostname('hourofcode.com') %>
  <% codedotorg = CDO.canonical_hostname('code.org') %>
  <% storedotcodedotorg = CDO.canonical_hostname('store.code.org') %>
```